ui/react-components/components/PatientSearch/PatientSearch.jsx

```diff
@@ -59,6 +65,11 @@ export const PatientSearch = (props) => {
                 label: intl.formatMessage({id: 'DROPDOWN_TYPE_TO_SEARCH_MESSAGE', defaultMessage: 'Type to search'}),
                 disabled: true
             }]);
+        } else{
+            setItems([{
+                label: intl.formatMessage({id: 'DROPDOWN_LOADING_MESSAGE', defaultMessage: 'Loading...'}),
```

Comment:
Can we refactor this if possible ?

separating this into a constants and set the value, we can also add the disabled key also incase if this disabled value is not changing in so many places.